shared/src/metabase/shared/util/i18n.clj

```diff
@@ -34,10 +34,22 @@
     :cljs
     `(js-i18n ~format-string ~@args)))
 
+(defmacro trun
```

Comment:
Not strictly related but I also want to use this, and it was missing, so I added it here too.